client/components/core/badge/index.tsx

```diff
@@ -0,0 +1,14 @@
+import { privateApis } from '@wordpress/components';
+import { __dangerousOptInToUnstableAPIsOnlyForCoreModules } from '@wordpress/private-apis';
+import React from 'react';
+
+const CoreBadge = ( { children }: { children: React.ReactNode } ) => {
```

Comment:
Also, should this be in a more central place (ie. a package) so that it can be imported across a8c?

Reply:
Thanks for the review, @ciampo! 

Do you suggest moving it to a package? I think we could move it here: `packages/core-badge` or something if you suggest. I haven't contributed to packages till now, but I'm assuming it is pretty straightforward (moving the component to packages in the same PR)

Reply:
Yeah, we haven't figured out how to approach any core package changes. We might want to fork packages separately, or the entire Gutenberg repo altogether for simplicity. In any case, moving a single component  to a separate standalone package sounds like the worst possible scenario - I think @mirka was researching that at some point and mentioned it's a really bad idea.

Reply:
For now I would recommend keeping it in `client/components`, i.e. Calypso-only.

Since this is just unlocking a private component, products that run in dependency-extracted environments (e.g. Jetpack, Woo) would be better to unlock it on their own from the globally available version. We can think about company-wide sharing when a non-dependency-extracted product want to use this too.